### ISSUE TITLE
S02.01-Exercise-Networking

### ISSUE THREAD

**Issue by funnygerman** [udacity/ud851-Sunshine#49]
Maybe the issue is sitting in front of the monitor, but for me following TODOs seem to be incomplete:
**TODO (8) Create a method that will get the user's preferred location and execute your new AsyncTask and call it loadWeatherData**
What should be the user's preferred location? I found it out only in your solution.

**TODO (6) Override the doInBackground method to perform your network requests**
The same, not really clear, what the student should do. At least not the stuff from the solution

**Comment by arjhun** [comment]
Yes this exercise really felt like suddenly they dropped the the student and I had to find all the methods myself, the previous exercises were much more guided, a big contrast!

**Comment by korsalc14** [comment]
Yes here is very interesting. 
TODO (6) and (8), student needs to understand Generic of AsyncTask< Param, Progress, Result> and how to use it, if student checks the solution. Otherwise student will get headache on next lesson S02.02.

**Comment by filograno** [comment]
I agree with @funnygerman and I think that this exercise must have more TODO than now in order to detail the entire exercise.

**Comment by shadyroid** [comment]
Thanks for your comments I thought that I'am fool and can't solve it with my own

**Comment by Smita07374** [comment]
Its so confusing
Did you guys manage to solve the exercise?
